src/components/validation/runner/mod.rs

```diff
@@ -339,7 +339,7 @@ impl Runner {
             //       I think we are going to need to setup distinct task sync logic potentially for each
             //       combination of Source/Sink + Resource direction Push/Pull
             if self.configuration.component_type == ComponentType::Sink {
-                tokio::time::sleep(Duration::from_secs(1)).await;
+                tokio::time::sleep(Duration::from_secs(5)).await;
```

Comment:
Why is sleep here needed? And why did we have to bump it for this PR?

Reply:
It's described a bit in the comment above. And, there is actually another sleep, about 40 lines up in this file. In this configuration, events longer to go through the tasks we've set up. In theory we are supposed to be able to not have any sleeps, because we have task syncrhonization, but there is a bug somewhere and/or we need to extend it. I have a separate task to investigate that. The sleeps are small, the overall time to run is still less than 15 seconds so it was enough that we are ok with proceeding with the sleeps temporarily.

Reply:
But it is something I'd like to resolve before we add the tests to CI

Reply:
resolved by merging in the fixes for this on master